### PR TITLE
Release v1.4.0

### PR DESCRIPTION
**Pull request by elenatorro**

### CODE REVIEW

File: src/renderer/viz/expressions.js [hunk 1]
```diff
@@ -451,20 +451,21 @@ import { Nin } from './expressions/belongs';
...
+import GreaterThan from './expressions/binary/GreaterThan';
+import GreaterThanOrEqualTo from './expressions/binary/GreaterThanOrEqualTo';
+import LessThan from './expressions/binary/LessThan';
+import LessThanOrEqualTo from './expressions/binary/LessThanOrEqualTo';
+import Mod from './expressions/binary/Mod';
+import Mul from './expressions/binary/Mul';
+import NotEquals from './expressions/binary/NotEquals';
+import Or from './expressions/binary/Or';
+import Pow from './expressions/binary/Pow';
+import Sub from './expressions/binary/Sub';
```
Comment (Jesus89) [inline src/renderer/viz/expressions.js]: :heart:

File: src/renderer/viz/expressions/binary/And.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * const viz = new carto.Viz(`
+ *   filter: $price < 30 and $category === 'fruit'  // Equivalent to and(lt($price, 30), eq($category, 'fruit'))
+ * `);
+ *
+ * @memberof carto.expressions
+ * @name and
+ * @function
+ * @api
+ */
+export default class Or extends BinaryOperation {
```
Comment (Jesus89) [inline src/renderer/viz/expressions/binary/And.js]: ~Or~ And

**Comment by Jesus89** [comment]
It seems that something is wrong with the browser tests

**Comment by elenatorro** [comment]
I know, taking a look to the logs, this is what we got:

Screenshot 2019-07-15 at 12 03 59

It seems there's a problem with one of the mapbox libraries, I'm taking a look. This is weird because this morning these tests where passing, so it might be a recent change.

**Comment by VictorVelarde** [comment]
Wait for me fir this release, I'm gonna probably add a small piece to docs that is required

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## UNRELEASED
+## [1.4.0] 2019-15-07
 
 ### Fixed
 - Fix `value`, `eval()` and `getLegendData()` for binary operations
```
Comment (VictorVelarde) [inline CHANGELOG.md]: Here I think we're missing the new options in GeoJSON source (mixed geometry types...)

**Pull request closed without merging**